sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/monitoring.properties

```
@@ -3,9 +3,9 @@ management.server.port=8081
 management.server.base-path=/
 management.endpoints.web.base-path=/
 management.endpoints.web.exposure.include=*
-management.endpoints.enabled-by-default=false
+management.endpoints.access.default=none
 # Healthcheck
-management.endpoint.health.enabled=true
+management.endpoint.health.access=read_only
```

In the notes you linked it's `read-only`.

```suggestion
management.endpoint.health.access=read-only
```

---

Imho the example in the docs [here](https://github.com/SDA-SE/sda-spring-boot-commons/blob/main/docs/starter-web.md#actuator-endpoints-tasks) needs an update as well.

---

You're absolutely right! This happened because I used IntelliJ's suggestions and didn’t notice the underscore (_). I was puzzled that both read_only and read-only seemed to work fine. I'll update it to match the documentation for consistency.